orangecontrib/text/widgets/owconcordance.py

```diff
@@ -0,0 +1,218 @@
+from collections import Counter
+from typing import Optional
+
+from AnyQt.QtCore import Qt, QAbstractTableModel, QSize
+from AnyQt.QtWidgets import QSizePolicy, QApplication, QTableView
+from AnyQt.QtGui import QBrush, QColor
+
+
+from Orange.data import Table
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.preprocess import Preprocessor, WordPunctTokenizer
+
+
+class IO:
+    DATA = 'Corpus'
```

Comment:
Not quite sure, but I think the main reason was that I prefer `IO.DATA` to `'Corpus'` throughout the codebase. I guess I just don't like strings as much, code completion works better and it's harder to make a mistake. Though — as @janezd suggested — we probably should move this 
higher in the hierarchy.